### DISCUSSION TITLE
Report designer fails on Mac OS 11.6.1

### DISCUSSION THREAD

**Discussion by colinsutton** [eclipse-birt/birt#764] [Q&A] [answered]
I built the 2012-12 eclipse on Mac OS 11.6. after refreshing my fork of birt-master - that seems OK:
<img width="959" alt="screenshot_553" src="https://user-images.githubusercontent.com/18058891/143796252-70beadfc-746c-49ef-9291-85e82032c650.png">

When I run the app with the hello_world.repdesign sample:
<img width="1380" alt="screenshot_554" src="https://user-images.githubusercontent.com/18058891/143796340-bb569eb0-2b2d-4d8b-860b-fb23e29668d5.png">

**Comment by wimjongman** [comment]
Colin, we need the log entries, stack trace, etc.. Please **attach** logs (not paste in text field)

**Comment by wimjongman** [comment]
You can export the error log from the error log view.

**Comment by colinsutton** [comment]
I don't see an error log view. I started with a new workspace, nothing happens when I select the report design perspective.
When I open the hello_world.rptdesign I see this:
<img width="1010" alt="screenshot_555" src="https://user-images.githubusercontent.com/18058891/143939490-d3cc3d39-b2f9-4012-bc35-e6b6a2da7a92.png">

[open hello_world.rptdesign error.txt](https://github.com/eclipse/birt/files/7620975/open.hello_world.rptdesign.error.txt)
:

**Comment by wimjongman** [comment]
Can you please do this CTRL+3  -> "copy configuration data to clipboard" -> paste in file -> attach

**Comment by colinsutton** [comment]
ctrl+3 didn't give any response, nor cmd+3. I couldn't find a shortcut that works on the Mac, here is the setup.log, will that do?
[setup.log](https://github.com/eclipse/birt/files/7622061/setup.log)

**Comment by colinsutton** [comment]
Found the error log. It seems I need an eclipse keyring? There is a \User\colin\.eclipse directory with the settings, but no .eclipse_keyring. Where does that come from? Did I miss a step?
[eclipse error.log](https://github.com/eclipse/birt/files/7622651/eclipse.error.log)

**Comment by wimjongman** [comment]
I see this in your log:

org.eclipse.birt.report.designer.ui.lib_4.7.0.v201706222054

What is that thing doing there?

**Comment by wimjongman** [comment]
It looks like you are processing an old target platform. Maybe there is a lingering setting in one of the poms somewhere for mac. I will take a look.

**Comment by wimjongman** [comment]
Colin, I have tested it on our mac and for me, it works like a charm. Almost out of the box, the only thing I needed to do is to add the mac specific bundles to the run configuration. 

![image](https://user-images.githubusercontent.com/180969/144049979-5e97219b-133d-4077-9d2a-57491a15cb8f.png)

**Comment by colinsutton** [comment]
How do I "add the mac specific bundles to the run configuration"?
I just updated my branch of birt-master, reinstalled eclipse in a new workarea and still have no report design perspective.
When I update eclipse it fails with:
<img width="754" alt="screenshot_556" src="https://user-images.githubusercontent.com/18058891/144124131-93dc41c2-9ceb-4325-9718-a3fab47ea5d1.png">

**Comment by wimjongman** [comment, marked as answer]
Colin, the Eclipse that you are installing is NOT a BIRT runtime environment. It is a BIRT development environment. You cannot run reports. Please take a look at the video. Section "Running BIRT with your changes" https://www.youtube.com/watch?v=FqfrG2I0AIw&t=540s
